snakefmt/parser/syntax.py

```diff
@@ -323,9 +330,17 @@ def in_brackets(self):
     def parse_params(self, snakefile: TokenIterator):
         cur_param = Parameter(self.token)
         prev_token = None
-
         while True:
             cur_param = self.process_token(cur_param, prev_token)
+            if (
+                self.token is not None
+                and sys.version_info >= (3, 12)
+                and self.token.type == tokenize.FSTRING_MIDDLE
+            ):
+                if self.token.string.endswith("}"):
+                    cur_param.value += "}"
```

Comment:
This line isn't touched by new tests but AFAICT it should be through your test covering #207?

Reply:
? It is if you look at https://app.codecov.io/gh/snakemake/snakefmt/pull/213?src=pr&el=tree#0c7239bd15aa6f28d7bd0459403810df-R340

Reply:
ah - github was flagging those two lines as uncovered, which indeed seemed impossible - resolved!